src/toolchain/toolchain.ts

```diff
@@ -492,7 +492,7 @@ export class SwiftToolchain {
         if (!base) {
             return undefined;
         }
-        return path.join(base, "usr/lib");
+        return `${path.join(base, "usr/lib")}:${path.join(base, "usr/lib/swift/macosx/testing")}`;
```

Comment:
`base` appears to be the _platform_ directory, but in this scenario the path should be relative to the toolchain